client/signup/config/flows.js

```diff
@@ -163,7 +163,7 @@ const Flows = {
 	 * @param {string} step Name of the step to be excluded.
 	 */
 	excludeStep( step ) {
-		step && Flows.excludedSteps.push( step );
+		step && Flows.excludedSteps.indexOf( step ) === -1 && Flows.excludedSteps.push( step );
```

Comment:
`excludedSteps` should probably be a `Set`. You don't have to do that refactor though, this is fine.